### ISSUE TITLE
Phasing rework reminder ..

### ISSUE THREAD

**Issue by slash-design** [jasongdove/LegionCore-7.3.5#98]
For later

Start at this point ?

https://github.com/TrinityCore/TrinityCore/commit/48ec2df81fa8f88cd32d7a79b587603aedbd89e0

**Comment by ReyDonovan** [comment]
yes, the phase system can be updated.... but this will require a separate branch and many fixes related to already implemented content

**Comment by slash-design** [comment]
Yeah, I've seen that and it gives me a headache -.-
